src/Clearinghouse.sol

```diff
@@ -328,6 +330,9 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
             sdai.approve(address(TRSRY), sdaiAmount);
             sdai.transfer(address(TRSRY), sdaiAmount);
         }
+
+        // Log the event.
+        emit Rebalance();
```

Comment:
It would be helpful to emit the amount in the event.

e.g. if daiBalance < maxFundAmount and DAI is being withdrawn from sDAI, emit fundAmount (positive number) as a parameter.

If daiBalance > maxFundAmount and DAI is being deposited into sDAI, emit defundAmount as a negative number in the parameter.

Without this, it would not be possible to know the amount that was transferred.

Reply:
ok, will do